test/integration/scheduler_perf/scheduler_perf.go

```diff
@@ -665,6 +661,44 @@ func (cpso createPodSetsOp) patchParams(w *workload) (realOp, error) {
 	return &cpso, (&cpso).isValid(true)
 }
 
+// deletePodsOp defines an op where previously created pods are deleted.
+// The test can block on the completion of this op before moving forward or
+// continue asynchronously.
+type deletePodsOp struct {
+	// Must be "deletePods".
+	Opcode operationCode
+	// Namespace the pods should be deleted from.
+	Namespace string
+	// Labels used to filter the pods to delete.
+	Labels map[string]string
```

Comment:
Actually, in which case will we need label selector? (isn't `Namespace` enough?)
If you're going to use it to delete Pods created by multiple `createPodsOp`, you need to mark `Namespace` as optional.

Reply:
Topology spreading plugin works only within a single namespace. We'll need the unschedulable pods blocked by topology spreading to be is the same namespace as the blocker pods that we'll need to delete.